composer.json

```diff
@@ -10,7 +10,9 @@
     },
     "require-dev": {
         "phpunit/phpunit": "~4.6",
-        "php-http/guzzle6-adapter": "^1.0"
+        "php-http/guzzle6-adapter": "^1.0",
+        "fabpot/php-cs-fixer": "^1.11",
+        "squizlabs/php_codesniffer": "^2.6"
```

Comment:
@Nyholm new PR https://github.com/mailgun/mailgun-php/pull/149 added to sort this case and provide a more accurate update